src/components/OutlinedCardList.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      marginTop: '10px',
+      border: '1px solid',
+      borderColor: theme.palette.divider,
+      borderRadius: '6px',
+    },
+    content: {
+      padding: '8px 12px',
+      textAlign: 'left',
+      display: 'flex',
+      flexDirection: 'column',
+    },
+    itemWrapper: {
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+    },
+    items: {
+      width: '100px',
+      color: theme.palette.text.primary,
+      margin: '5px',
+    },
+  }),
+);
+
+interface CardProps {
+  numbers: number[];
+  labels: string[];
+  unit: string[];
+  primary: boolean;
+  rightIcon?: JSX.Element;
+}
+
+export default function OutlinedColCard(props: CardProps) {
+  const classes = useStyles();
+
+  const getLabeledNumber = (
+    key: number,
+    number: number,
+    label: string,
+    unit: string,
+    primary: boolean,
+    rightIcon: JSX.Element,
+  ) => {
+    return (
+      <div key={key} className={classes.itemWrapper}>
+        <div className={classes.items}>
+          <Typography variant="h4">{label}</Typography>
+          <Typography variant="h3" color={primary ? 'primary' : 'inherit'}>
+            {number} {unit}
+          </Typography>
+        </div>
+        {rightIcon}
+      </div>
+    );
+  };
+
+  const rightIcon = props.rightIcon ? props.rightIcon : <></>;
```

Comment:
does the styling break if we do null here instead of `<></>`? If not, returning null might be clearer than an empty element (if people aren't familiar with this in jsx)